move_to_object:
  ros__parameters:
    robot_number: 1

    object_name: 'redpringles'